posthog/tasks/test/test_process_scheduled_changes.py

```diff
@@ -0,0 +1,91 @@
+from datetime import datetime, timedelta, timezone
+from posthog.models import ScheduledChange, FeatureFlag
+from posthog.test.base import APIBaseTest
+from posthog.tasks.process_scheduled_changes import process_scheduled_changes
+
+
+class TestProcessScheduledChanges(APIBaseTest):
```

Comment:
I think we could do with a test or two to cover something like

there are four scheduled changes 

* one due in the past, 
* one due in the past and already executed
* one due exactly now, 
* and one due in the future

when we run the task the one in the future doesn't execute, the already executed one doesn't execute, and the other two do

(simple enough to assert that from reading the code now but if we add more features to this or have to change it in a rush during an incident then it's good to have the main case covered)

Reply:
Yep good idea, added the test now!